plan/resolve_indices.go

```diff
@@ -203,15 +203,19 @@ func (p *Show) ResolveIndices() {
 	}
 }
 
-// ResolveIndices implements Plan interface.
-func (p *basePlan) ResolveIndices() {
+func (p *physicalSchemaProducer) ResolveIndices() {
```

Comment:
Can we remove this plan's function? Then we can use basePlan's `ResolveIndices` directly.

Reply:
basePlan no longer contains schema field.

Reply:
Got it.